### DISCUSSION TITLE
ESLint plugin absolute path resolution issue.

### DISCUSSION THREAD

**Discussion by bobbytreed** [webpack/webpack#16787] [Q&A]
# Bug report

**What is the current behavior?**
ESLint tries to import a eslint-config-prettier module from node_modules using an absolute path. All of the paths in __webpack_require__.c are relative paths.  The lookup fails.

![image](https://user-images.githubusercontent.com/10874547/223264744-a02d796c-2bb8-4ae0-ab49-e8468a1bf91b.png)

And I get the following error

![image](https://user-images.githubusercontent.com/10874547/223265059-43d44694-7127-4f98-9b6a-7952d12be324.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
The absolute path should be resolved appropriately.  

I have tried, aliases, scopes, and a few resolve hacks, but nothing seems to stick.  I accept that this could be a very easy configuration of which I'm not familiar and thus need to research more.  Any help in pointing me to the correct documentation is an acceptable closure of the issue as well.

**Other relevant information:**
webpack version:  5
Node.js version: 18
Operating System: Windows 11
Additional tools: ESLint

**Comment by alexander-akait** [comment]
Yeah, but we can't use absolte path, what do you try to solve, I will help, can you provide example of the problem and what do you try to solve - github repo/code/etc, feel free to feedback

**Reply by bobbytreed** [reply]
So, I'm linting code within markdown files, and providing AI QuickFixes, etc.

In my LintJS Code function

![image](https://user-images.githubusercontent.com/10874547/226345381-7329e8d6-413c-4d35-a0cd-ef651a279b38.png)

I pass in the lintConfig which is a eslintrc.json run through JSON.parse()

I provide  the current working directory to ESLint as well, but there is this line in eslintrc.json

![image](https://user-images.githubusercontent.com/10874547/226345865-8451ec46-36fb-45e6-8e02-3195e66454f8.png)

This tells ESLint to load the eslint-prettier plugin which, as I'm sure you know, removes rules that are duplicative in prettier (more/less).

when ESLint (after webpack) tries to go load the eslint-prettier plugin, ESLint reconstructs an absolute path (for some unknown reason) and tries to get the module (plugin) that way. 

And of course it fails.

Unless I can get ES Lint to fix or tell me what the issue is inside there code, I need to provide some type of alias so that when ES Lint tries to look up 

![image](https://user-images.githubusercontent.com/10874547/226346773-bde66a0f-281d-4c79-bddf-eef1e8756af0.png)

I can tell it where it actually lives.

**Reply by bobbytreed** [reply]
![image](https://user-images.githubusercontent.com/10874547/226347460-791bff17-448a-4683-a8cc-da8e9d6fb7eb.png)

I assume that I could be overlooking a simple solution, but obviously I cannot find it.

I appreciate any feedback you can provide

**Reply by bobbytreed** [reply]
Also, looks like bunding ESLint is not supported at the moment.

https://github.com/eslint/eslint/issues/16966#issuecomment-1461218677

**Reply by alexander-akait** [reply]
Yeah, maybe you can found a solution based on https://github.com/eslint/eslint/issues/16966#issuecomment-1461218677 comment
